ui/config/deprecation-workflow.js

```diff
@@ -5,6 +5,8 @@ self.deprecationWorkflow.config = {
     { handler: 'throw', matchId: 'ember-inflector.globals' },
     { handler: 'throw', matchId: 'ember-runtime.deprecate-copy-copyable' },
     { handler: 'throw', matchId: 'ember-console.deprecate-logger' },
-    { handler: 'silence', matchId: 'ember-component.send-action' },
```

Comment:
TIL about this file

Reply:
From this addon! https://github.com/mixonic/ember-cli-deprecation-workflow

Super helpful for triaging deprecations. I don't think you're supposed to commit it like this, but I'm gonna try it out. My main fear is deprecations related to `str.pluralize()` creeping back in. This makes it a runtime error now.